explorer/site-config/hca-dcp/dev/config.ts

```diff
@@ -13,7 +13,13 @@ import HcaLogo from "images/hca-logo.png";
 const config: SiteConfig = {
   redirectRootToPath: "/explore/projects",
   datasources: {
-    catalog: "dcp2",
+    defaultListParams: {
+      catalog: "dcp2",
```

Comment:
Can you add `CATALOG_DCP2` and `PAGINATION_PAGE_SIZE` constants local to this file?